src/main/java/org/kohsuke/github/GHRepository.java

```diff
@@ -988,6 +997,18 @@ public void enableIssueTracker(boolean v) throws IOException {
         edit("has_issues", String.valueOf(v));
     }
 
+    /**
+     * Enables or disables projects for this repository.
+     *
+     * @param v
+     *            the v
+     * @throws IOException
+     *             the io exception
```

Comment:
This will probably fail formatting, but we need links to the api docs. 
```suggestion
     *             the io exception
     * @see https://developer.github.com/v3/repos/#parameters-4
```